python/pyspark/sql/readwriter.py

```diff
@@ -1225,7 +1226,6 @@ def overwrite(self, condition):
         Overwrite rows matching the given filter condition with the contents of the data frame in
         the output table.
         """
-        condition = _to_java_column(column)
```

Comment:
It could not resolve `column`, this seems like a bug

Reply:
@zero323 does that make sense? looks like this was added in your PR

Reply:
I think it is a copy/paste error:
```python
    @since(3.1)
    def overwrite(self, condition):
        """
        Overwrite rows matching the given filter condition with the contents of the data frame in
        the output table.
        """
        condition = _to_java_column(column)
        self._jwriter.overwrite(condition)
```
Above is the original function, I don't know where the column could come from. cc @zero323 @HyukjinKwon

Reply:
Yeah, I think it was a mistake.

Reply:
Indeed. Thanks for catching this @Fokko

Reply:
Thanks for letting me know